patches/patches.json

```diff
@@ -8,6 +8,9 @@
   "v13.12.0": [
     "node.v13.12.0.cpp.patch"
   ],
+  "v12.18.4": [
+    "node.v12.18.4.cpp.patch"
```

Comment:
@n1ru4l If the patch is the same of 12.18.3, what about just point the 12.18.3 here?

Reply:
I just replaced `12.18.3` completely with `12.18.4` (this is a different branch than the one we are using :))

Reply:
Ok ignore my comment so :)